services/autocomplete.py

```diff
@@ -0,0 +1,16 @@
+import urllib.request
+import json
+def message3(var):    

```

Comment:
@wimpykid26  I would highly suggest removing this file altogether and re-writing this code in JS. Create a new file similar to https://github.com/osdc/osdc-bot/blob/master/services/jokeService.js and add a block in chatbot.js (see https://github.com/osdc/osdc-bot/blob/master/chatbot.js#L105-L107) to simply call that method from the 1st file!